### PR TITLE
Cleaning up the central event content structure

### PR DESCRIPTION
**Pull request by jeongeun**
#### PR description:  
Cleaning up the central event content structures.
- For improving clarity, moving extend commands to the right after the corresponding output commands (PSet). 
- Clean up unused parts and Remove duplicated import lines. 
- Unify the Modifier styles. 
1 central event content file (`Configuration/EventContent/python/EventContent_cff.py` ) and
1 additional files also changed (`EventFilter/ScalersRawToDigi/python/Scalers_EventContent_cff.py`).

#### PR validation:
Event Content comparison check was also done and there is no change with these updates.
Tested in CMSSW_11_1_X, the basic test all passed in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html).

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: Configuration/EventContent/python/EventContent_cff.py [hunk 1]
```diff
@@ -286,6 +455,57 @@
     outputCommands = cms.untracked.vstring('drop *'),
     splitLevel = cms.untracked.int32(0),
 )
+FEVTSIMEventContent.outputCommands.extend(RAWEventContent.outputCommands)
```
Comment (perrotta) [inline Configuration/EventContent/python/EventContent_cff.py:458]: This shold be better defined extending the FEVTEventContent
Then, whatever already added on top of RAWEventContent to obtain the FEVTEventContent can be removed from the list below

File: Configuration/EventContent/python/EventContent_cff.py [hunk 2]
```diff
@@ -576,254 +708,79 @@
...
+        'drop *_ak8GenJetsNoNu_*_*',
+        'drop *_genCandidatesForMET_*_*',
+        'drop *_genParticlesForMETAllVisible_*_*',
+        'drop *_genMetCalo_*_*',
+        'drop *_genMetCaloAndNonPrompt_*_*',
+        'drop *_genMetTrue_*_*',
+        'drop *_genMetIC5GenJs_*_*')
 )
 
-def SwapKeepAndDrop(l):
```
Comment (perrotta) [inline Configuration/EventContent/python/EventContent_cff.py:770]: This function could be maintained, for possible future usages.
Maybe you can just add it back on top of this file, right after alll import's

File: Configuration/EventContent/python/EventContent_cff.py [hunk 3]
```diff
@@ -834,32 +791,26 @@ def SwapKeepAndDrop(l):
...
     outputCommands = cms.untracked.vstring('drop *'),
     eventAutoFlushCompressedSize=cms.untracked.int32(20*1024*1024),
     compressionAlgorithm=cms.untracked.string("LZMA"),
     compressionLevel=cms.untracked.int32(4)
 )
-
-RAWMINIAODEventContent.outputCommands.extend(MicroEventContent.outputCommands)
-RAWMINIAODEventContent.outputCommands.extend(L1TriggerRAW.outputCommands)
-RAWMINIAODEventContent.outputCommands.extend(HLTriggerRAW.outputCommands)
 RAWMINIAODSIMEventContent.outputCommands.extend(MicroEventContentMC.outputCommands)
```
Comment (perrotta) [inline Configuration/EventContent/python/EventContent_cff.py:807]: RAWMINIAODSIMEventContent should be better made by extending the RAWMINIAODEventContent

File: Configuration/EventContent/python/EventContent_cff.py [hunk 4]
```diff
@@ -885,39 +835,31 @@ def SwapKeepAndDrop(l):
 for _entry in [MINIAODEventContent, MINIAODSIMEventContent]:
     fastSim.toModify(_entry, outputCommands = _entry.outputCommands + fastSimEC.dropPatTrigger)
 
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
```
Comment (perrotta) [inline Configuration/EventContent/python/EventContent_cff.py:838]: All these phase2, gem, HI modifications should be better moved right below the corresponding definitions of the event content, as it is done already for hgcal and pps (for example)

File: Configuration/EventContent/python/EventContent_cff.py [hunk 5]
```diff
@@ -885,39 +835,31 @@ def SwapKeepAndDrop(l):
 for _entry in [MINIAODEventContent, MINIAODSIMEventContent]:
     fastSim.toModify(_entry, outputCommands = _entry.outputCommands + fastSimEC.dropPatTrigger)
```
Comment (perrotta) [inline Configuration/EventContent/python/EventContent_cff.py:836]: Ok for letting all FastSim modifications concentrated here

**Review by perrotta — COMMENTED** [review]
Since you are cleaning, I would also move [line 59](https://github.com/cms-sw/cmssw/blob/c3add5e1f270d511958a5cc4a19a4fe23fae6270/Configuration/EventContent/python/EventContent_cff.py#L59) into a category on its own right above HLT, i.e.
```PYTHON
#
#
# L1
#
#
from L1Trigger.Configuration.L1Trigger_EventContent_cff import *
```

File: Configuration/EventContent/python/EventContent_cff.py [hunk 6]
```diff
@@ -354,6 +369,13 @@
 RECOSIMEventContent.outputCommands.extend(SimFastTimingRECO.outputCommands)
 RECOSIMEventContent.outputCommands.extend(SimGeneralRECO.outputCommands)
 RECOSIMEventContent.outputCommands.extend(MEtoEDMConverterRECO.outputCommands)
+
+from Configuration.Eras.Modifier_phase2_timing_layer_cff import phase2_timing_layer
+from RecoLocalFastTime.Configuration.RecoLocalFastTime_EventContent_cff import RecoLocalFastTimeFEVT, RecoLocalFastTimeRECO, RecoLocalFastTimeAOD
```
Comment (perrotta) [inline Configuration/EventContent/python/EventContent_cff.py:374]: This line can be moved above, where also the event contents for hgcal are imported (and a simple ```import *``` can be used overall)

File: Configuration/EventContent/python/EventContent_cff.py [hunk 7]
```diff
@@ -435,10 +462,37 @@
...
 FEVTEventContent.outputCommands.extend(EITopPAGEventContent.outputCommands)
 
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_phase2_timing_layer_cff import phase2_timing_layer
+from RecoLocalFastTime.Configuration.RecoLocalFastTime_EventContent_cff import RecoLocalFastTimeFEVT, RecoLocalFastTimeRECO, RecoLocalFastTimeAOD
+from RecoMTD.Configuration.RecoMTD_EventContent_cff import RecoMTDFEVT, RecoMTDRECO, RecoMTDAOD
```
Comment (perrotta) [inline Configuration/EventContent/python/EventContent_cff.py:472]: This line can be moved above, where also the event contents for hgcal are imported (and a simple ```import *``` can be used overall)

File: Configuration/EventContent/python/EventContent_cff.py [hunk 8]
```diff
@@ -435,10 +462,37 @@
 FEVTEventContent.outputCommands.extend(CommonEventContent.outputCommands)
 FEVTEventContent.outputCommands.extend(EITopPAGEventContent.outputCommands)
 
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_phase2_timing_layer_cff import phase2_timing_layer
+from RecoLocalFastTime.Configuration.RecoLocalFastTime_EventContent_cff import RecoLocalFastTimeFEVT, RecoLocalFastTimeRECO, RecoLocalFastTimeAOD
```
Comment (perrotta) [inline Configuration/EventContent/python/EventContent_cff.py:471]: not needed, duplicate

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Central event contents reorganized and reordered as planned
- Actual contents remain unchanged
- jenkins tests pass and show no differences

**Comment by silviodonato** [comment]
+operations

**Comment by silviodonato** [comment]
+1

**Pull request merged**